src/utils/domainMetadata.ts

```diff
@@ -4,8 +4,29 @@ SPDX-License-Identifier: LGPL-3.0-only
 */
 import { Environment } from "@buildwithsygma/sygma-sdk-core"
 
+export type ResourceMetadata = {
+  caip19: string
+  symbol: string
+  decimals: number
+}
+
+export type EnvironmentResourcesMetadata = {
+  [key: number]: ResourceMetadata[]
+}
+
 export type DomainMetadata = {
```

Comment:
should we add explorer URL?

Reply:
will do actually, so we avoid using this from this env var that we define on cloudflare